cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -73,14 +72,6 @@
       cronjob at:'*/5 * * * *', do:deploy_dir('bin', 'snapshot')
     end
 
-    if node.chef_environment == 'levelbuilder'
-      cronjob at:'0 20 * * 1-5', do:deploy_dir('bin', 'cron', 'commit_content')
-      cronjob at:'18 23 * * 1-5', do:deploy_dir('bin', 'cron', 'commit_content')
```

Comment:
the purpose of this line is to avoid merge conflicts during DTL when incoming changes touch files which have uncommitted local changes on the levelbuilder machine. so, I think your best options are:
1. also disable `deploy_to_levelbuilder` on line 71, or
2. re-enable line 78 here, and if levelbuilder content does need to be scooped, do a more precise cherry-pick from what you see in the levelbuilder branch when you want to scoop changes in from levelbuilder.

I would suggest starting with (1), and if we need to DTL at some point during lockdown, it will be easy to add back lines 71 and 78 to transition into option 2.

Reply:
cc @code-dot-org/ed-programs-tools in case anyone would like to advocate for keeping DTLs going during lockdown. of course, levelbuilder would only be getting updated from changes merged to staging, not staging-next.

Reply:
nit: if you go with option 1, I think it would be a little easier to leave the if clause and just comment out the lines you don't want, so that it is easier to restore bit by bit later if needed.

Reply:
My memory is that curriculum (level builder) content that needs to be released during the lockdown needs to be hand staged, committed, and pushed on the levelbuilder server since there may be unrelated content changes that we would not want to blanket scoop on level builder.

Reply:
Aside from hand selecting the level builder content to commit/push, I think the Developer of the Day will use the commands in the Developer of the Day script to commit/push/merge content during the lockdown period, and then we would re-enable our content pipeline by reverting this entire Pull Request.